pdns/packethandler.cc

```diff
@@ -1546,8 +1546,7 @@ DNSPacket *PacketHandler::doQuestion(DNSPacket *p)
     if(doSigs)
       addRRSigs(d_dk, B, authSet, r->getRRS());
       
-    r->wrapup(); // needed for inserting in cache
```

Comment:
was this comment wrong?

Reply:
There is no need to do it here getString() will warpup when necessary https://github.com/powerdns/pdns/blob/master/pdns/dnspacket.cc#L63